text/0000-module-resolution.md

```diff
@@ -244,14 +250,74 @@ This type of resolution tells the resolver to find an npm package with that name
 ```json
 {
     "modules": [
-       { "npm" : "@ui/components" }     
+       { 
+           "npm" : "@ui/components"
+       }     
     ]
 }
 ```
 
 `NpmModuleRecord` must contain `npm` key.
 
-## Algorithm
+## Module resolution and exposure via NPM packages
+
+So far we have covered how are we going to resolve the modules, however we would want to be able to expose a subset of those modules 
+for other developers in the form of an npm package.
+
+### Exposing modules
+In order to expose a module via npm package, it must be explicity declared in the `expose` array property.
+
+```json
+{
+    "modules": [
+       { 
+           "dir" : "src/modules"
+       }     
+    ],
+    "expose": [
```

Comment:
maybe use "export" instead

Reply:
I support the `expose` proposal. It maps with the `isExposed` property used in the `metadata.xml` on the platform.